WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -528,7 +528,7 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
             return
         }
 
-        PostListEditorPresenter.handle(post: post, in: self)
+        post.hasVersionConflict() ? PostListConflictResolver.handle(post: post, in: self) : PostListEditorPresenter.handle(post: post, in: self)
```

Comment:
Do you think the Post Lists really need to be aware of the version conflict?

I guess that it can delegate that to some entity to present the post and this entity will do any other work that needs to be done.

Reply:
I'm not sure what this comment means - it's already shifting the presentation work to the PostListEditorPresenter or the PostListConflictResolver?